staging/istio/templates/zipkin-svc.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
How/where is zipkin installed? I wonder if this file belongs here.

Reply:
Its very confusing the way it is implemented in Istio. Bear with me! The actual application is Jaeger. Jaeger use to be part of Istio installation but no more. So now it is deployed as an independent addon. However, Istio still depend on a service named `zipkin` (don't ask me why the service name is `zipkin`) to connect to Jaeger. As Jaeger is separated from Istio, it has stopped creating this service. Hence, we have to create this service for istio to talk to Jaeger

Reply:
lgtm.

can we put this as a comment somewhere?